src/validation.cpp

```diff
@@ -7286,3 +7286,80 @@ class CMainCleanup
         mapBlockIndex.clear();
     }
 } instance_of_cmaincleanup;
+
+std::pair<Ranks, size_t> ANVRanks(
+        const std::vector<CAmount>& anvs,
+        int height,
+        const Consensus::Params& params)
+{
+    assert(height >= 0);
+    assert(prefviewdb != nullptr);
+
+    static size_t max_embassador_lottery = 0;
+    referral::AddressANVs entrants;
+
+    // unlikely that the candidates grew over 50% since last time.
+    auto reserve_size = max_embassador_lottery * 1.5;
```

Comment:
Nitpick: it's s/embassador/ambassador (unless this is on purpose and I'm not familiar with the jargon).